File: New Password.py
s = "abcdefghijklmnopqrstuvwxyz"

a, b = input().split()

for i in range(0,int(b)):
    print(s[i], end="")

diff = int(a)-int(b)
j = 0

for i in range(0,diff,1):
    print(s[j], end="")
    j+=1

    if j == int(b):
        j=0

File: New Password.cpp
#include<bits/stdc++.h>
using namespace std;

int main()
{
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    
    string s = "abcdefghijklmnopqrstuvwxyz";
    int a,b;

    while(cin>>a>>b)
    {
        for(int i=0; i<b; i++)
        {
            cout<<s[i];
        }
        
        int diff = a-b;
        
        for(int i=0; i<b && diff>0; i++, diff--)
        {
            cout<<s[i];
            if(diff-1 != 0 && i+1 == b) i=-1;
        }
        
        cout<<endl;
        
    }
}
